storage/dataflux/worksteal.go

```diff
@@ -138,7 +138,7 @@ func (w *worker) doWorkstealListing(ctx context.Context) error {
 		// Worker is active when it finds work in range channel.
 		if w.status == idle {
 			if len(w.lister.ranges) == 0 {
-				time.Sleep(time.Millisecond * time.Duration(sleepDurationWhenIdle))
+				time.Sleep(time.Millisecond * sleepDurationWhenIdle)
```

Comment:
Converting to milliseconds should be part of the definition of the `sleepDurationWhenIdle`.